stubs/WTForms/wtforms/i18n.pyi

```diff
@@ -0,0 +1,30 @@
+from collections.abc import Callable, Iterable
+from gettext import GNUTranslations
+from typing import Protocol, TypeVar, overload
+
+_T = TypeVar("_T")
+
+class _SupportsUgettextAndUngettext(Protocol):
+    def ugettext(self, __string: str) -> str: ...
+    def ungettext(self, __singular: str, __plural: str, __n: int) -> str: ...
+
+def messages_path() -> str: ...
+def get_builtin_gnu_translations(languages: Iterable[str] | None = None) -> GNUTranslations: ...
+@overload
+def get_translations(
+    languages: Iterable[str] | None = None, getter: Callable[[Iterable[str]], GNUTranslations] = ...
+) -> GNUTranslations: ...
+@overload
```

Comment:
This seems redundant with the next overload.

Reply:
This allows for `get_translations(getter=...)` instead of `get_translations(None, ...)`. I would prefer not having to write two overloads either, but this is the only way to cover both call signatures.

Reply:
Ah right, this is python/typing#1475. You have to really stare at these overloads sometimes.